src/app/components/PageLayoutWrapper/index.tsx

```diff
@@ -114,7 +115,7 @@ const PageLayoutWrapper = ({
       <ThemeProvider service={service} variant={variant}>
         {!isNextJs && <ServiceWorkerContainer />}
         <ManifestContainer />
-        <WebVitals pageType={pageType} />
+        {!isErrorPage && pageData && <WebVitals pageType={pageType} />}
```

Comment:
Can we now remove this?
```suggestion
        {!isErrorPage && <WebVitals pageType={pageType} />}
```